EventAggregator.Tests/SomeMessageHandler.cs

```diff
@@ -53,4 +57,117 @@ public void Handle(SomeMessage2 message)
             _eventsTrapped.Add(message);
         }
     }
+
+    public class SomeMessageHandler4 : IListener<SomeMessage>
+    {
+        public void Handle(SomeMessage message)
+        {
+            throw new InvalidOperationException("");
+        }
+    }
+
+    public class FakeException : Exception { }
+
+    public class SomeMessageHandler5 : IListener<SomeMessage>
+    {
+        public void Handle(SomeMessage message)
+        {
+            throw new TargetInvocationException(new FakeException());

```

Comment:
Reading the above tests, the TargetInvocationException with inner TargetInvocationException confused me. Could change this exception type to make it clear that the test is just not dropping inner exceptions?